src/lib/schematics/utils/devkit-utils/config.ts

```diff
@@ -49,6 +49,9 @@ export interface Workspace {
 export interface Project {
   name: string;
 
+  /** Application prefix. */
+  prefix: string;
```

Comment:
Unless this is part of the Angular CLI's actual project schema, we can't add this extra property here without also making it non-enumerable because we reserialize this object and write it to disk when we're e.g. adding a theme.

Reply:
Yes, this is in the actual schema. I pulled this code from official component schematic - https://github.com/angular/angular-cli/blob/master/packages/schematics/angular/component/index.ts#L116-L125